### PR TITLE
Add support for DITA Open Toolkit.

### PR DESCRIPTION
**Pull request by evandervalk**
This change adds support for installing the repository as a [DITA Open Toolkit](https://www.dita-ot.org/download) plugin.

When merged, it allows installing the repository as a zip archive or through a remote zip archive using the Github archive URL.

### CODE REVIEW

**Review by brocelot — APPROVED** [review]

File: LICENSE.md
Comment (devatwork) [inline LICENSE.md]: NIT: Should be a range: 2019-2022

File: README.md
Comment (devatwork) [inline README.md]: NIT: should be an ordered lists. Otherwise the steps may be executed in arbitrary order which is not the case.

**Review by devatwork — CHANGES_REQUESTED** [review]
LGTM! 2 nits, after which this PR may be merged.

File: LICENSE.md
Comment (evandervalk) [reply LICENSE.md]: Updated the range from 2019 to 2022.

File: README.md
Comment (evandervalk) [reply README.md]: Updated the unordered list to an ordered list.

**Review by devatwork — APPROVED** [review]
LGTM! Thanks! :shipit:

**Pull request merged**
